.travis.yml

```diff
@@ -23,17 +23,22 @@ jobs:
     - stage: unit tests
       install: skip
       jdk: oraclejdk11
-      script: mvn test -Dtest=com.napier.sem.AppTest
+      script: mvn test -Dtest=com.napier.sem.MyTest
     - stage: integration tests
       jdk: oraclejdk11
       script: echo "Integration Tests"
```

Comment:
this should be doing more than echoing integration tests rn, i think we should maybe switch these two around.